com.unity.perception/Runtime/GroundTruth/Labelers/KeyPointLabeler.cs

```diff
@@ -72,7 +73,8 @@ public KeyPointLabeler(IdLabelConfig config, KeyPointTemplate template)
         /// <summary>
         /// Array of animation pose labels which map animation clip times to ground truth pose labels.
         /// </summary>
-        public AnimationPoseLabel[] poseStateConfigs;
+        [FormerlySerializedAs("poseStateConfigs")]
```

Comment:
I'm not sure if we'll need this attribute since the package hasn't released yet

Reply:
Good catch. I meant to remove that but put it there temporarily so my test files weren't broken